test/quantization/test_quantize_script.py

```diff
@@ -1015,6 +1016,22 @@ def forward(self, x):
                    .check("aten::dequantize") \
                    .run(model.graph)
 
+    def test_finalize_no_extra_dequantize(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 3).float()
+
+            def forward(self, x):
+                y = x.size(0)
+                x = self.conv(x)
+                return (x.size(0) + y) * x
```

Comment:
Why does this case have extra dequant? Is it due to x.size()?

Reply:
Yes